test/libsolidity/semanticTests/viaYul/require.sol

```diff
@@ -12,17 +12,25 @@ contract C {
 		x = true;
 		require(true);
 	}
-	/* Not properly supported by test system yet
 	function f2(bool a) public pure returns (bool x) {
 		x = a;
 		string memory message;
+		message = "fancy message!";
 		require(a, message);
```

Comment:
Can you keep a case with `string memory message; require(a, message);` to test empty strings?